charts/consul/values.yaml

```
@@ -627,7 +627,7 @@ global:
   # The name (and tag) of the consul-dataplane Docker image used for the
   # connect-injected sidecar proxies and mesh, terminating, and ingress gateways.
   # @default: hashicorp/consul-dataplane:<latest supported version>
-  imageConsulDataplane: docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.3-dev
+  imageConsulDataplane: docker.io/hashicorppreview/consul-dataplane:1.3-dev-7cb9b2276b9bec30f5a583faf0e55356d41fa750
```

Let's change this back to test the recent build containing an additional patch
```suggestion
  imageConsulDataplane: docker.io/hashicorppreview/consul-dataplane:1.3-dev
```